addons/packages/fluxcd-helm-controller/0.17.2/package.yaml

```
@@ -5,7 +5,7 @@ metadata:
 spec:
   refName: helm-controller.fluxcd.community.tanzu.vmware.com
   version: 0.17.2
-  releasedAt: 2022-05-01T16:40:55Z
+  releasedAt: "2022-05-01T16:40:55"
```

```suggestion
  releasedAt: 2022-05-01T16:40:55Z
```

The releasedAt needs to be in this format, otherwise the package repository will not reconcile. It throws the following error:
```
parsing time "2022-05-01T16:40:55" as "2006-01-02T15:04:05Z07:00": cannot parse "" as "Z07:00"
```
If I recall correctly, we had this problem when the 0.17.2 version was originally added. The change was only made here in the PR. To correctly fix this problem, the package yaml needs to be updated in the `package-for-helm-controller` source repository. With the fix made there, when this package is updated in the future and `vendir sync` is run, we will get the correctly updated file. This will be the same fix for the readme file as well.

---

I have resolved this. The issue was the assets under `v0.17.2` were still having the old format and caused this issue.